community/unsafe/src/main/java/org/neo4j/unsafe/impl/internal/dragons/MemoryManager.java

```diff
@@ -32,8 +32,7 @@
     /**
      * The amount of memory, in bytes, to grab in each Slab.
      */
-    private static final long GRAB_SIZE = Integer.getInteger(
-            MemoryManager.class.getName() + ".GRAB_SIZE", 512 * 1024 ); // 512 KiB
+    private static final long GRAB_SIZE = FeatureToggles.getInteger( "GRAB_SIZE", 512 * 1024 ); // 512 KiB
```

Comment:
That is the case with how the pattern `FeatureToggles` encapsulates is used today as well. Nothing about that changes with the change of this line.